### PR TITLE
enable openai v1 in load.py

### PR DESCRIPTION
**Pull request by wenzhe-log10**
* load.py supports both openai v0 and v1
* llm.openai and examples support openai v1 only. 

Relaxed the openai dependency to `<2.0.0`. 
Users should install openai by them self and log10 installation should leave the existing openai version.
- [ ] need to test the pip installation 

Plan to release log10 version `0.5.0` (currently `0.4.6`).

### CODE REVIEW

File: examples/logging/chatcompletion_async_vs_sync.py
Comment (wenzhe-log10) [inline examples/logging/chatcompletion_async_vs_sync.py:1]: removed chatcompletion_async_vs_sync.py due to cannot import openai twice with v1
`ImportError: PyO3 modules may only be initialized once per interpreter process`

**Comment by wenzhe-log10** [comment]
Added the tags. 
e.g.
```python
from log10.load import Log10_OpenAI as OpenAI
client = OpenAI(tags=["load_v1_test"])
completion = client.chat.completions.create(
    model="gpt-3.5-turbo",
    messages=[{"role": "user", "content": "Hello world"}],
)
print(completion)
```

@nqn Please review.

File: examples/logging/chatcompletion_sync.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import openai
+
+from log10.load import log10, log10_session
+
+
+log10(openai, DEBUG_=True, USE_ASYNC_=False)
```
Comment (nqn) [inline examples/logging/chatcompletion_sync.py:6]: This should be with the new log10 openai, right?

File: examples/logging/chatcompletion_sync.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+with log10_session():
```
Comment (nqn) [inline examples/logging/chatcompletion_sync.py:7]: Will this keep working with the new setup?

File: log10/load.py [hunk 3]
```diff
@@ -361,42 +388,57 @@ def set_sync_log_text(USE_ASYNC=True):
...
     Keyword arguments:
     module -- the module to be intercepted (e.g. openai)
     DEBUG_ -- whether to show log10 related debug statements via python logging (default False)
     USE_ASYNC_ -- whether to run in async mode (default True)
 
+    Openai V1 example:
+    Example:
+        >>> from log10.load import log10
+        >>> import openai
+        >>> log10(openai)
```
Comment (nqn) [inline log10/load.py:402]: We want this to happen within the class, so we can obsolete the library wrapper, right?

File: examples/logging/chatcompletion_sync.py (hunk 2 above)
Comment (wenzhe-log10) [reply examples/logging/chatcompletion_sync.py]: yes, tested and it works.

File: log10/load.py (hunk 3 above)
Comment (wenzhe-log10) [reply log10/load.py]: remove load.log10 v1 examples here and use load.OpenAI instead (has examples there)

File: examples/logging/chatcompletion_sync.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+from log10.load import OpenAI, log10_session
+
+
+client = OpenAI()
+
+with log10_session(tags=["log10-io/examples"]):
+    client = OpenAI()
```
Comment (nqn) [inline examples/logging/chatcompletion_sync.py:7]: Nit: do you need this one here?

Comment (wenzhe-log10) [reply examples/logging/chatcompletion_sync.py]: good catch. that's duplicated.

**Review by nqn — APPROVED** [review]

**Pull request merged**